app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
```

Comment:
Code Style issue: According to the checklist, variable names should be descriptive. While 'comfort_class' and 'clean_mark' are somewhat descriptive, they could be improved. Consider renaming 'comfort_class' to 'comfort_level' and 'clean_mark' to 'cleanliness_level', which more accurately describe what these variables represent.